integration/spark/src/test/scala/org/apache/carbondata/integration/spark/testsuite/dataload/TestLoadDataWithSingleQuotechar.scala

```diff
@@ -0,0 +1,60 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.carbondata.integration.spark.testsuite.dataload
+
+import org.apache.spark.sql.{DataFrame, Row}
+import org.apache.spark.sql.common.util.CarbonHiveContext._
+import org.apache.spark.sql.common.util.QueryTest
+import org.scalatest.BeforeAndAfterAll
+
+/**
+ * Test Class for data loading when there is single quote in fact data
+ *
+ */
+class TestLoadDataWithSingleQuotechar extends QueryTest with BeforeAndAfterAll {
+  override def beforeAll {
+    sql("DROP TABLE IF EXISTS carbontable")
+    sql(
+      "CREATE TABLE carbontable (id Int, name String) STORED BY 'carbondata'")
+  }
+
+  test("test data loading with single quote char") {
+    try {
+      sql(
+        "LOAD DATA LOCAL INPATH './src/test/resources/dataWithSingleQuote.csv' INTO TABLE " +
+          "carbontable OPTIONS('DELIMITER'= ',')")
+      checkAnswer(
+        sql("SELECT * from carbontable"),
+        Seq(Row("Tom",1),
+          Row("Tony\n3,Lily",2),
+          Row("Games\"",4),
+          Row("prival\"\n6,\"hello\"",5)
+        )
```

Comment:
1) previous version we were supporting multiline=true/false option in data load, but after moving to univocity parser, that option is discontinued and defaulted to multiline=true
2) What happens if this multiline is split across blocks, as textinputformat will skip first line of current block and will read one extra new line of next block. It might end up reading, same line twice. - can be checked once and can be tracked as separate issue.
3) future we can support option to set multiline=false for univocity parser, similar to hive.So this behavior of carbon is fine.
4) I think multiline+parse till quote+delimiter as expected behaviour needs to be mentioned in IDG document.